pkgs/applications/editors/emacs-modes/manual-packages.nix

```
@@ -110,6 +110,27 @@
   org-mac-link =
     callPackage ./org-mac-link { };
 
+  org-roam = melpaBuild rec {
+    pname = "org-roam";
+    version = "0.1.2";
+    src = pkgs.fetchFromGitHub {
+      owner = "jethrokuan";
+      repo = "org-roam";
+      rev = "v${version}";
+      sha256 = "1p8bhj09s0iyb3fcjibsl1p61wiac17sn1w7hwm1wqrcydj8h8hx";
+    };
+    recipe = pkgs.writeText "recipe" ''
+      (org-roam
+       :repo "jethrokuan/org-roam"
+       :fetcher github
+       :files ("*.el"))
+    '';
+    packageRequires = [ dash f s async ];
```

for when 1.0.0 lands:

```suggestion
    packageRequires = [ dash f s emacsql emacsql-sqlite ];
```

---

@jethrokuan If the package is pulled from melpa all those bit's are automagic.